### ISSUE TITLE
Automatically Pin Cases as the "Pinned Content" in the Chat

### ISSUE THREAD

**Issue by xehu** [StanfordHCI/bang#470]
Before the case functionality was finished, I had been providing the links to the cases in the reading period, and then providing the links again in the 'pinned content' area of the chat, so that users can refer to them again during the round, something like this:

<img width="1058" alt="Screen Shot 2020-01-01 at 5 08 28 PM" src="https://user-images.githubusercontent.com/28793641/71646728-731be000-2cb9-11ea-987f-a2f539e95437.png">

However, now that the cases are randomly-generated, I can no longer use hard-coded links to provide the cases. **Is it possible to automatically put a link to part 1 and part 2 of the cases via the pinned content feature?**

In other words, the logic should go as follows: if there are cases defined, generate a reading period for each case (this is already done). Then, put the same cases in a link so that they are accessible in the 'pinned content' section.

**Comment by dtatarenkov** [comment]
in prod
